Mixpanel/SessionMetadata.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Metadata.swift
+//  Mixpanel
+//
+//  Created by Yarden Eitan on 10/24/17.
+//  Copyright © 2017 Mixpanel. All rights reserved.
+//
+
+import Foundation
+
+class SessionMetadata {
+    var eventsCounter: UInt64 = 0
+    var peopleCounter: UInt64 = 0
+    var sessionID: UInt64 = 0
+    var sessionStartEpoch: UInt64 = 0
+
+    func applicationDidBecomeActive() {
```

Comment:
do you think we need to synchronize this? an event could be tracked before the app goes into the foreground again and counter would be 0